CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 ### App Center Distribute
 
 * **[Feature]** Remove the download manager task if the download doesn't start within 10 seconds.
-* **[Feature]** Replace install a new release from deprecated intent action [ACTION_INSTALL_PACKAGE](https://developer.android.com/reference/android/content/Intent#ACTION_INSTALL_PACKAGE) to `PackageInstaller` API.
+* **[Feature]** Replace installing a new release using the deprecated intent action [ACTION_INSTALL_PACKAGE](https://developer.android.com/reference/android/content/Intent#ACTION_INSTALL_PACKAGE) with the `PackageInstaller` API.
```

Comment:
Why are these changes here? Please revert all changes from the main code and merge PR with the base branch.